tests/HardwareTestHarness.md

```diff
@@ -0,0 +1,181 @@
+Hardware Test Harness
+=====================
+
+There is an implementation of the hardware testing design which is a small 4in x 4in board with positions for
+two Wemos D! Mini ESP8266 boards, a breadboard area and a number of positions for peripherals. The schematics
+are in [schematic](Test-harness-schematic-v1.pdf) and a rendering of the board is ![Board Render](Test-Harness-Render-V1.png). 
```

Comment:
Just realized that `Test-Harness-Render-V1.png` is missing but I vaguely remember @pjsg posted an image in a comment to some test-related issue. I failed to dig that one out, though.

Reply:
Found it: https://github.com/nodemcu/nodemcu-firmware/pull/3324#issuecomment-735282277. I'll add it to `dev` now.